### PR TITLE
add tests for DS_STORE and macos meta information in tar.gz/zip

### PR DESCRIPTION
**Pull request by KutuzovGregory**

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/csv/CSVUploadTest.java [hunk 1]
```diff
@@ -140,6 +124,50 @@ public void testGzCsvBinaryUpload() throws Exception {
         checkBinaryFileUpload(entityName, metricName, csvPath);
     }
 
+    /* #2966 */
+    @Test
+    public void testDSStoreFileInTarGz() throws Exception {
+        String entityName = ENTITY_PREFIX + "-9";
+        String metricName = METRIC_PREFIX + "-9";
+
+        File csvPath = resolvePath(RESOURCE_DIR + File.separator + name.getMethodName() + ".tar.gz");
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/csv/CSVUploadTest.java]: why you call it Path?

Comment (KutuzovGregory) [inline src/test/java/com/axibase/tsd/api/method/csv/CSVUploadTest.java]: From File javadoc:
> An abstract representation of file and directory pathnames.

**Comment by Sild** [comment]
ok

**Comment by unrealwork** [comment]
ok

**Comment by rmakulov** [comment]
ok

**Pull request merged**